File: examples/pb-net-grpc/Server_VB/Startup.vb
Imports Microsoft.AspNetCore.Builder
Imports Microsoft.AspNetCore.Hosting
Imports Microsoft.Extensions.DependencyInjection
Imports ProtoBuf.Grpc.Server

Public Class Startup
    Public Sub ConfigureServices(services As IServiceCollection)
        services.AddCodeFirstGrpc()
    End Sub

#Disable Warning IDE0060 ' Remove unused parameter
    Public Sub Configure(app As IApplicationBuilder, env As IWebHostEnvironment)
#Enable Warning IDE0060 ' Remove unused parameter
        app.UseRouting()

        app.UseEndpoints(
            Sub(endpoints)
                endpoints.MapGrpcService(Of MyCalculator)()
            End Sub)
        End Sub
End Class

File: examples/wcf-port/TraderSys/src/TraderSys.Portfolios/Startup.cs
using Microsoft.AspNetCore.Builder;
using Microsoft.AspNetCore.Hosting;
using Microsoft.Extensions.DependencyInjection;
using Microsoft.Extensions.Hosting;
using ProtoBuf.Grpc.Server;
using TraderSys.PortfolioData;
using TraderSys.Portfolios.Services;


namespace TraderSys.Portfolios
{
    public class Startup
    {
        public void ConfigureServices(IServiceCollection services)
        {
            services.AddScoped<IPortfolioRepository, PortfolioRepository>();
            services.AddCodeFirstGrpc();
        }

        public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
        {
            if (env.IsDevelopment())
            {
                app.UseDeveloperExceptionPage();
            }

            app.UseRouting();

            app.UseEndpoints(endpoints =>
            {
                endpoints.MapGrpcService<PortfolioService>();
            });
        }
    }
}